snapshot/store/http_store.go

```diff
@@ -46,11 +46,21 @@ type httpStore struct {
 	client  Client
 }
 
-func (s *httpStore) OpenForRead(name string) (io.ReadCloser, error) {
+func getTTLValue(resp *http.Response) *TTLValue {
+	ttlk := http.CanonicalHeaderKey("x-ton-expires")
+	expire := resp.Header.Get(ttlk)
+	ttl, err := time.Parse(time.RFC1123, expire)
```

Comment:
If we do this we should also make the format string a const value in store.go and add as part of the TTLConfig as like TTLFormat

Reply:
Done.